shim.js

```diff
@@ -1,22 +1,23 @@
-if (typeof __dirname === 'undefined') global.__dirname = '/'
-if (typeof __filename === 'undefined') global.__filename = ''
-// Needed so that 'stream-http' chooses the right default protocol.
-global.location = {
-  protocol: 'file:',
-}
+require('node-libs-react-native/globals')
```

Comment:
de-duplicate
look at what globals.js define and dont repeat in shim.js

Reply:
yes i actually did this exactly. 
global defines location, process and some other stuff which was removed from shim.js

i've just left atob/btoa because we're using a different polyfill library than node-libs-react-native
i suspected there was some reason, so i've kept `Base64` usage

Reply:
there's no point in double base64 packages